x-pack/plugins/inference/common/connectors.ts

```diff
@@ -11,16 +11,14 @@ export enum InferenceConnectorType {
   Gemini = '.gemini',
 }
 
+export const allSupportedConnectorTypes = Object.values(InferenceConnectorType);
```

Comment:
nit: This doesn't appear to be used outside of this file. Is there a need to export it?

Reply:
Not anymore you're right. Fixed in https://github.com/elastic/kibana/pull/191069/commits/a8d38ea287ac0a38ac0154a191599eac1cb33230